### PR TITLE
Makes sneaky rodents able to avoid traps

### PR DESCRIPTION
**Pull request by IamVelcroboy**
## About the PR
- Reduces walk speed of mice and hamsters so they are able to avoid traps by being sneaky.   

## Why / Balance
I feel this is balanced by being slow and allows for some more interesting gameplay for mice players

## Technical details
n/a

## Media
n/a

## Requirements
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an in-game showcase.

## Breaking changes

**Changelog**
:cl: Velcroboy
- tweak: Sneaky mice and hamsters can avoid traps by walking very, very slow

### CODE REVIEW

**Comment by Radezolid** [comment]
Oh nice, no longer being completely blocked out of the kitchen with 5 mouse traps! I love it.

**Comment by IamVelcroboy** [comment]
ooo, forgot comments...

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 1]
```diff
@@ -1903,7 +1903,7 @@
       state: slug
       sprite: Mobs/Animals/slug.rsi
   - type: Carriable #DeltaV
-    freeHandsRequired: 1
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: untouch

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 2]
```diff
@@ -3359,7 +3359,7 @@
   - type: HTN
     rootTask:
       task: RuminantCompound
-
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: untouch

**Review by MilonPL — CHANGES_REQUESTED** [review]
unfuck whitespaces and it's good to go

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml (hunk 1 above)
Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: This just happens sometimes. You can't readd these spaces and have them recognized. At least I've never been ale to

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml (hunk 2 above)
Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: yeah, there's nothing I can really do about it

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml (hunk 1 above)
Comment (MilonPL) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: > This just happens sometimes. You can't readd these spaces and have them recognized. At least I've never been ale to

You could either revert the commit or revert to the old version of the file with `git log` and `git checkout [hash] -- path/to/file`, then only commit the lines that you want changed.

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: Seems like a lot of work to preserve some necessary spaces

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: Doesn't work. No matter what I do, those lines are always different
image

Comment (perryprog) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: Personally the way I'd do it (which I find a bit more intuitive) is
```
# Note the trailing tilde following the hash. In this case it'd be git checkout -p 7116a7b~ ...
git checkout -p (commit hash with bad whitespace)~ (file(s) with bad whitespace)
```
And you'd then go through and enter `y` for parts of the commit that have bad whitespace and `n` for parts that were fine. You'd then might see some warnings about you having added in trailing whitespace, which is what you want. (This in general is how I prefer to selectively undo only certain line changes of a commit.) Once done you can commit as normal.

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: So, I tried this. Seemed to work. I committed and pushed the file again and I don't see any change

Comment (MilonPL) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: > So, I tried this. Seemed to work. I committed and pushed the file again and I don't see any change

Wrong branch Velcro! https://github.com/DeltaV-Station/Delta-v/compare/master...IamVelcroboy:Delta-v-rebase:Snooky_Rojents

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: Hey...look at that

**Review by MilonPL — APPROVED** [review]

**Pull request merged**